guillotina/db/storages/pg.py

```diff
@@ -235,6 +235,20 @@ def _wrap_return_count(txt):
 ''')
 
 
+def restart_conn_on_exception(f):
+    async def decorator(self: 'PostgresqlStorage', *args, **kwargs):
+        try:
+            return await f(self, *args, **kwargs)
+        except asyncpg.exceptions.PostgresConnectionError as ex:
+            await self._check_bad_connection(ex)
+            raise
+        except asyncpg.exceptions.InterfaceError as ex:
```

Comment:
I think this can be combined with above right?

`except (asyncpg.exceptions.PostgresConnectionError, asyncpg.exceptions.InterfaceError) as ex`

Reply:
Ups, you're right!